Lib/test/test_pdb.py

```diff
@@ -3935,6 +3935,25 @@ def test_readrc_homedir(self):
                     f.write("invalid")
                 self.assertEqual(pdb.Pdb().rcLines[0], "invalid")
 
+    def test_readrc_current_dir(self):
+        with os_helper.temp_cwd() as cwd:
+            rc_path = os.path.join(cwd, ".pdbrc")
+            with open(rc_path, "w") as f:
+                f.write("invalid")
+            self.assertEqual(pdb.Pdb().rcLines[0], "invalid")
+            self.assertEqual(len(pdb.Pdb().rcLines), 1)
+
+    def test_readrc_home_twice(self):
```

Comment:
Let's rename the function name to be something more descriptive. `test_readrc_cwd_is_home` maybe?

Reply:
done